Source/CarthageKit/FrameworkExtensions.swift

```diff
@@ -268,20 +269,22 @@ extension URL {
 	internal func swiftHeaderURL() -> URL? {
 		let headersURL = self.appendingPathComponent("Headers", isDirectory: true).resolvingSymlinksInPath()
 		let dirContents = try? FileManager.default.contentsOfDirectory(at: headersURL, includingPropertiesForKeys: [], options: [])
-		return dirContents?.filter { $0.absoluteString.contains("-Swift.h") }.first
+		return dirContents?.first { $0.absoluteString.contains("-Swift.h") }
```

Comment:
Note that this should even make the code run faster as it can cut filtering early (once the first object is found).